lib/hosts.py

```diff
@@ -64,12 +64,21 @@
 PROD_ACCOUNT = "451493790433"
 PROD_DOMAIN = "theboss.io"
 PROD_LAMBDA_BUCKET = "boss-lambda-prod-env"
-PROD_LAMBDA_SERVER = "52.55.121.6"
+PROD_LAMBDA_SERVER = "54.210.116.141"
```

Comment:
Should these IPs be EC2 names that are then resolved? (so we don't have to change the file when we rebuild) (and elastic IP might also work to limit file changes)

Reply:
Yes, we should switch to host names down the road. I'm not familiar with setting that up, so didn't incorporate that into this PR/task.